.ci/build-kit/install_and_test.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+set -e
+
+cmake \
+    -B build \
+    -S "$EXT_MOUNT/source" \
+    -G Ninja \
+    -DBUILD_TESTING=ON \
+    -DCMAKE_BUILD_TYPE=Debug \
+    -DCMAKE_INSTALL_PREFIX="$WORKSPACE_PATH/dist"
+
+ninja -j$(nproc) -C build install
+
+ninja -j$(nproc) -C build tests/test
```

Comment:
```suggestion
ninja -j$(nproc) -C build test
```

Unit tests should be "registered" by CTest and ran by the target `test`